docs/source/visualisation/kedro-viz_visualisation.md

```diff
@@ -37,7 +37,9 @@ The command opens a browser tab to serve the visualisation at `http://127.0.0.1:
 
 You should see the following:
 
-![](../meta/images/pipeline_visualisation.png)
+```{eval-rst}
+.. thumbnail:: ../meta/images/pipeline_visualisation.png
+```
```

Comment:
Maybe let's rework this into MyST

````suggestion
```{thumbnail} ../meta/images/pipeline_visualisation.png
```
````

(haven't tried this syntax yet, please check https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html)

Reply:
I've changed the syntax and removed the image_config setting